addons/mail/static/src/models/thread/thread.js

```diff
@@ -1701,6 +1804,16 @@ function factory(dependencies) {
             compute: '_computeNeedactionMessages',
             dependencies: ['messages'],
         }),
+        /**
+         * Id of the message under which the new message separator should be placed.
+         */
```

Comment:
I'm very lame for finding good name so ..."messageIdUnderWhichNewMessageSeparatorIsPlaced" but seems long ..

Reply:
This suggestion is acceptable, obviously it would be best if we found a shorter name that conveys the same meaning (we can think about it) but until then I prefer a longer and clear version like this than unclear short version.

Reply:
`messageBeforeNewMessageSeparatorId` is a little bit shorter